def raise_exception
  puts 'antes'
  raise 'Ocorreu um erro'
  puts 'depois'
end

raise_exception
